src/postal/message.clj

```diff
@@ -102,13 +102,36 @@
     (doto (javax.mail.internet.MimeBodyPart.)
       (.setContent (eval-multipart part)))))
 
+(defn bytes? [x]
+  (if (nil? x)
+    false
+    (= (Class/forName "[B")
+       (.getClass x))))
+
+(defmulti make-mime-part (fn [part]
+                           (if (bytes? (:content part))
+                             :byte-array
+                             :url)))
+
+(defmethod make-mime-part :byte-array
+  [part]
+  (let [headers (doto (InternetHeaders.)
+                  (.setHeader "Content-Transfer-Encoding" "base64"))]
+    (javax.mail.internet.MimeBodyPart. headers
+                                       (:content part))))
```

Comment:
Assuming it's a byte-array not previously encoded, maybe the library could be doing the encoding itself.

Didn't want to add a new dependency to do that though, what's your take on it?